src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String OPERATION_SUPPLY = "supply";
+    private static final String OPERATION_BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+                BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+            int[] totals = new int[2]; // totals[0] - supplyTotal, totals[1] - buyTotal
```

Comment:
The current implementation uses a custom format for generating statistics, which may introduce potential issues in terms of code scalability and maintainability. Instead of returning an array of integers, it would be more beneficial to handle the statistics within a single method, eliminating the need to pass or parse the data elsewhere. By encapsulating the processing logic within the method, developers can achieve better code organization and reduce dependencies on magic numbers or position-based interpretations. This approach promotes readability and facilitates future modifications to the statistics generation process.